db/migrate/20160830233154_add_port_ranges_to_load_balancer_listener.rb

```diff
@@ -0,0 +1,8 @@
+class AddPortRangesToLoadBalancerListener < ActiveRecord::Migration[5.0]
+  def change
+    change_table :load_balancer_listeners do |t|
+      t.column :load_balancer_port_range, :int4range, :default => nil, :null => true
+      t.column :instance_port_range, :int4range, :default => nil, :null => true
```

Comment:
@selmanj Minor, but I think `:default => nil, :null => true` are both default for a column, so no need to specify in the migration.